tests/prod/prod_sanity_suite_test.go

```diff
@@ -186,6 +226,16 @@ func TestCloudRouter2WanCreateConnection_DIGP(t *testing.T) {
 	terraform.InitAndApply(t, terraformOptions)
 	output := terraform.Output(t, terraformOptions, "wan_connection_id")
 	assert.NotNil(t, output)
+	terraform.Apply(t, terraformOptions)
```

Comment:
This shouldn't need to be here because it is already being done earlier in method with `InitAndApply`. Why is the redundant `Apply` in place?